### PR TITLE
added navbar to website

### LINKED ISSUE #8: Create Navigation Bar/Header .
Create NavBar/Header of the website refer to the original website: www.technonjr.org.

Things to keep in mind:

1. The menu should be responsive that is it should work well on both mobiles and desktop screens.
2. Good coding practice should be followed.
3. The Pull request need not be in a single commit.

### PR DESCRIPTION
**Pull request by sahuvivek2000**
## Explanation

Fixes #8: The PR includes Navbar.html which includes responsive navbar with logo

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum:".)
- [x] The PR explanation includes the words "Fixes #bugnum:" (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR is made from a branch that's **not** called "master".
- [x] The PR is made from a branch that is up-to-date with "master".
- [x] The PR's branch is based on "master" and not on any other branch.

### CODE REVIEW

**Review by jkugsiya — CHANGES_REQUESTED** [review]
The desktop version of the website is fine but when switching to the mobile version the options disappear, that is expected behaviour though, but those options should be available in a hamburger menu instead.

**Review by jkugsiya — CHANGES_REQUESTED** [review]
Also, implement the list options when hovering some of the tabs as in the original website www.technonjr.org

**Comment by sahuvivek2000** [comment]
working on it

**Review by jkugsiya — CHANGES_REQUESTED** [review]
When we switch to the mobile version, the logo doesn't come up in the center it is slightly left from the center.

**Review by jkugsiya — CHANGES_REQUESTED** [review]
Also keep all the images under the images folder already made.

**Comment by jkugsiya** [comment]
All the changes LGTM but follow this https://github.com/jkugsiya/TechnoIndiaNJR/pull/12#pullrequestreview-671736975 don't create a new image folder. The images folder is already present use that to keep all the images.

**Comment by jkugsiya** [comment]
Also follow this: https://github.com/jkugsiya/TechnoIndiaNJR/pull/12#pullrequestreview-671736547

**Comment by sahuvivek2000** [comment]
there is no image folder

**Comment by jkugsiya** [comment]
> there is no image folder

Fetch upstream and you will see there is one images folder in there
Follow [CONTRIBUTING.md](https://github.com/jkugsiya/TechnoIndiaNJR/blob/master/CONTRIBUTING.md) to see how to fetch and merge upstream

File: navbar.html [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    <script src='index.js'></script>
+</head>
+
+<body>
+
+    <!-- navbar -->
+    <nav class="navbar navbar-expand-lg navbar-dark">
+        <div class="container-fluid">
+            <a class="navbar-brand" href="#">
+                <img src="./image/Techno-Logo_new.png" alt="" class="d-inline-block align-text-top">
```
Comment (jkugsiya) [inline navbar.html:24]:
```suggestion
                <img src="./images/Techno-Logo_new.png" alt="" class="d-inline-block align-text-top">
```

**Comment by jkugsiya** [comment]
Thanks, @sahuvivek2000 LGTM. Approving changes and merging.

**Pull request merged**